src/backend/distributed/commands/function.c

```diff
@@ -213,6 +220,33 @@ create_distributed_function(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * ErrorIfAnyNodeDoesNotHaveMetadataSynced throws error if any
+ * of the worker nodes does not have the metadata.
+ */
+static void
+ErrorIfAnyNodeDoesNotHaveMetadataSynced(void)
+{
+	List *workerNodeList =
+		ActivePrimaryNonCoordinatorNodeList(ShareLock);
+
+	WorkerNode *workerNode = NULL;
+	foreach_ptr(workerNode, workerNodeList)
+	{
+		if (!workerNode->hasMetadata)
+		{
+			ereport(ERROR, (errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+							errmsg("cannot process the distributed function "
+								   "since the node %s:%d does not have metadata "
+								   "synced ", workerNode->workerName,
+								   workerNode->workerPort),
+							errhint("To sync the metadata execute: "
```

Comment:
We should update the hint whatever name we give here: https://github.com/citusdata/citus/pull/5556